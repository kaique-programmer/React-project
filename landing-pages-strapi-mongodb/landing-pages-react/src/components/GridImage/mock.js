export default {
  background: false,
  title: 'My grid',
  description:
    'Atque doloribus nobis laudantium esse ut, non commodi maxime distinctio veritatis unde',
  grid: [
    {
      altText: 'Teste 1',
      srcImage: 'http://source.unsplash.com/random/360x360?r=1',
    },
    {
      altText: 'Teste 2',
      srcImage: 'http://source.unsplash.com/random/360x360?r=2',
    },
    {
      altText: 'Teste 3',
      srcImage: 'http://source.unsplash.com/random/360x360?r=3',
    },
    {
      altText: 'Teste 4',
      srcImage: 'http://source.unsplash.com/random/360x360?r=4',
    },
    {
      altText: 'Teste 5',
      srcImage: 'http://source.unsplash.com/random/360x360?r=5',
    },
    {
      altText: 'Teste 6',
      srcImage: 'http://source.unsplash.com/random/360x360?r=6',
    },
  ],
};
