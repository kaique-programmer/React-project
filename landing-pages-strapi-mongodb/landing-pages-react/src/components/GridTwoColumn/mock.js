export default {
  title: 'GridTwoColumn',
  text: `Lorem, ipsum dolor sit amet consectetur adipisicing elit.
           Ipsum nostrum accusamus id alias quos! Reiciendis a rerum,
           esse exercitationem, animi nulla nobis natus id et earum obcaecati asperiores!
           Inventore, similique.`,
  srcImage: 'assets/images/javascript.svg',
}
