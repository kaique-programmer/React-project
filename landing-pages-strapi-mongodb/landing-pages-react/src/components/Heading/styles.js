import styled, { css } from 'styled-components';

export const Title = styled.h1`
  ${({ theme }) => css`
    color: ${light ? theme.colors.white : theme.colors.primaryBackground }
  `}
`;
