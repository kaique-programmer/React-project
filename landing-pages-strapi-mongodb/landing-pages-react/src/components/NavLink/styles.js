import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    display: flex;
    background: ${theme.colors.white};
    flex-flow: row wrap;

    @media ${theme.media.lteMedium} {
      flex-flow: column wrap;
      align-content: center;
    }
  `}
`;
