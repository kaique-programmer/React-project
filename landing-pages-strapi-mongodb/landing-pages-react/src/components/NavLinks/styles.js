import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    display: flex;
    background: #fff;
    flex-flow: row wrap;

    @media ${theme.media.lteMedium} {
      flex-flow: column wrap;
      align-content: center;
    }
  `}
`;
