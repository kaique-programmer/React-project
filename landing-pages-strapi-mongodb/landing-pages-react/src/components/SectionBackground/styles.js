import styled, { css } from 'styled-components';

const containerBackgroundActivate = (theme) => css`
  background: ${theme.colors.white};
  color: ${theme.colors.primaryBackground};
`;

export const Container = styled.div`
  ${({ theme, background }) => css`
    background: ${theme.colors.white};
    color: ${theme.colors.primaryBackground};
    ${background && containerBackgroundActivate(theme)};
    min-height: 100vh;
    display: flex;
    align-items: center;
  `}
`;
