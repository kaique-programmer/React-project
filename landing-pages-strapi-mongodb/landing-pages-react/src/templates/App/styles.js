import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ props }) => css`
    background: ${props.background};
  `}
`;
