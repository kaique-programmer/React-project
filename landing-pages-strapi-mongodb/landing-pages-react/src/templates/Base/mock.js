import mockLinks from '../../components/NavLinks/mock';

export default {
  links: mockLinks,
  logoData: {
    text: 'Logo',
    link: '#',
  },
  footerHtml: '<p>Footer test</p>'
}
