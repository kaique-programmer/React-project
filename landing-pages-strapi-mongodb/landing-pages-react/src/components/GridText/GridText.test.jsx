import React from 'react';
import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/render-theme';
import {GridText} from '.';

import mock from './mock';

describe('<GridText />', () => {
  it('should render', () => {
    renderTheme(<GridText>Children</GridText>);
    expect(screen.getByRole('heading')).toBeInTheDocument();
  });

  it('should render without background', () => {
    const { container } = renderTheme(
      <GridText {...mock} background={undefined} />,
    );
    expect(container).toMatchSnapshot();
  });
});
