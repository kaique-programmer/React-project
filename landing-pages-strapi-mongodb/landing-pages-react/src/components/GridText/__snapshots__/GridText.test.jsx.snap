// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridText /> should render without background 1`] = `
<div>
  <div
    class=""
  >
    <h1 />
  </div>
</div>
`;
